### PR TITLE
move further executables/scripts to /usr

### PR DESCRIPTION
**Pull request by mtomaschewski**
https://github.com/openSUSE/sysconfig/issues/33

### CODE REVIEW

File: scripts/Makefile.am [hunk 1]
```diff
@@ -30,12 +30,10 @@ substitute_vars	= $(AM_V_GEN)sed \
 	@$(substitute_vars) < $< > $@.tmp && mv $@.tmp $@
 
 install-data-hook:
-	$(MKDIR_P) $(DESTDIR)$(sysconfig_network_scriptsdir)
```
Comment (cfconrad) [inline scripts/Makefile.am:33]: Where do we need `sysconfig_network_scriptsdir` then, Maybe we can also cleanup `configure.ac` and the definition above in this file?

File: sysconfig.spec.in [hunk 2]
```diff
@@ -79,7 +87,7 @@ provided settings like DNS or NIS into system files.
 autoreconf -fvi
 CFLAGS="%{optflags} -fPIC" LDFLAGS="-pie" \
 %configure --prefix=/ \
-            --sbindir=/sbin \
+            --sbindir=%{_sbindir} \
             --libdir=/%{_lib} \
```
Comment (cfconrad) [inline sysconfig.spec.in:91]: maybe: 
```suggestion
            --libdir=%{_lib} \
```

**Review by cfconrad — APPROVED** [review]
LGTM, only nitty questions from my side...

File: scripts/Makefile.am (hunk 1 above)
Comment (mtomaschewski) [reply scripts/Makefile.am:33]: The move does not invalidate it completely, there may be still user scripts/hooks inside or worse,
user scripts may source function.netconfig from this location or we want to mention it in docs.
I've moved the link creation to the spec file to make it easier to conditionally not install the link
if we decide to break compatibility to existing customer scripts (a no-go for SLES, but ...).

File: sysconfig.spec.in (hunk 2 above)
Comment (mtomaschewski) [reply sysconfig.spec.in]: Adjusted to `--libdir=%{_libdir}`.

**Pull request merged**